projects/1.20/assets/enderite-cj-edition/enderite/lang/zh_cn.json

```
@@ -0,0 +1,107 @@
+{
+  "creativemodetab.enderite.enderite_tab": "末影合金",
+
+  "enchantment.enderite.night_vision": "夜视",
+  "enchantment.enderite.no_gravity": "无重力",
+  "enchantment.enderite.void_floating": "虚空漂浮",
+  "enchantment.enderite.night_vision.desc": "赋予玩家夜视效果",
+  "enchantment.enderite.no_gravity.desc": "使物品失重",
+  "enchantment.enderite.void_floating.desc": "保护物品不受虚空影响",
+
+  "item.enderite.endercrystal_shard": "末影水晶碎片",
+  "item.enderite.ender_dust": "末影尘",
+  "item.enderite.raw_enderite": "粗末影合金",
+  "item.enderite.enderite_scrap": "末影合金碎片",
+  "item.enderite.enderite_ingot": "末影合金锭",
+  "item.enderite.enderite_nugget": "末影合金粒",
+
+  "item.enderite.enderite_helmet": "末影合金头盔",
+  "item.enderite.enderite_chestplate": "末影合金胸甲",
+  "item.enderite.enderite_elytra_chestplate": "末影合金鞘翅盔甲",
+  "item.enderite.enderite_leggings": "末影合金护腿",
+  "item.enderite.enderite_boots": "末影合金靴子",
+
+  "item.enderite.enderite_sword": "末影合金剑",
+  "item.enderite.enderite_pickaxe": "末影合金稿",
+  "item.enderite.enderite_axe": "末影合金斧",
+  "item.enderite.enderite_shovel": "末影合金铲",
```

```suggestion
  "item.enderite.enderite_shovel": "末影合金锹",
```
建议还是先对着bot的diff术语提示，把你翻译的所有内容都查一遍吧
已经是第四次提示“原版术语翻译问题请全文通查”了，错误太多真的没法逐一指出